INV4/pallet-ips/src/benchmarking.rs

```diff
@@ -0,0 +1,62 @@
+//! Benchmarks for IPS Pallet
+#![cfg(feature = "runtime-benchmarks")]
+
+pub use super::*;
+use frame_benchmarking::{benchmarks, impl_benchmark_test_suite, vec, whitelisted_caller};
+use frame_system::RawOrigin;
+use primitives::{
+    InvArchLicenses,
+    OneOrPercent::{One, ZeroPoint},
+};
+use sp_runtime::Percent;
+
+macro_rules! percent {
+    ($x:expr) => {
+        ZeroPoint(Percent::from_percent($x))
+    };
+}
+
+benchmarks! {
+    where_clause {
+        where T: ipl::Config<Licenses = InvArchLicenses>
+    }
+    create_ips {
+        let caller: T::AccountId = whitelisted_caller();
+        let metadata: Vec<u8> = vec![1];
+        let data: Vec<<T as ipf::Config>::IpfId> = vec![];
+        let license = InvArchLicenses::GPLv3;
+    }: _(RawOrigin::Signed(caller), metadata, data, true, None, license, percent!(50), One, false)
+
+    destroy {
+        let s in 0 .. 100;
+        let caller: T::AccountId = whitelisted_caller();
+    }: _(RawOrigin::Signed(caller), T::IpsId::from(s))
+
+    append {
+        let s in 0 .. 100;
+        let caller: T::AccountId = whitelisted_caller();
+    }: _(RawOrigin::Signed(caller), T::IpsId::from(0u32), Default::default(), Some(vec![s.try_into().unwrap()]))
```

Comment:
Appending an IPS in another IPSwithout cresting IPS either one of them, also missing benchmarks for appending IPF inside IPS

Reply:
Check

Reply:
Creating IPS done, minting IPF inside IPS unresolved yet, as in create_ips above.